include/deal.II/lac/petsc_matrix_base.h

```diff
@@ -344,6 +344,13 @@ namespace PETScWrappers
      */
     virtual ~MatrixBase() override;
 
+    /**
+     * This method assignes the PETSc Mat to the instance of the class.
+     *
+     */
+    void
+    assign_petsc_matrix(Mat A);
```

Comment:
Also copy the statement about ownership from the corresponding function in the block matrix case above.